### PR TITLE
add sentry probe

### PR DESCRIPTION
**Pull request by qdequele**

### CODE REVIEW

File: meilisearch-http/src/main.rs [hunk 1]
```diff
@@ -27,6 +36,12 @@ async fn main() -> Result<(), MainError> {
                         .into(),
                 );
             }
+
+            #[cfg(feature = "sentry")]
+            if !opt.no_analytics {
+                sentry::integrations::panic::register_panic_handler();
+                sentry::integrations::env_logger::init(None, Default::default());
+            }
```
Comment (Kerollmops) [inline meilisearch-http/src/main.rs:44]: `cfgs` works on if statements? Is it real?

Comment (qdequele) [reply meilisearch-http/src/main.rs:44]: I ran it with the `--no-default-features` and without, and in both cases, it works as expected

Comment (Kerollmops) [reply meilisearch-http/src/main.rs:44]: Hum, waht version of rust are you using?

Comment (qdequele) [reply meilisearch-http/src/main.rs:44]: 1.43.0

**Pull request merged**